app/pkg/analysis/cputemp_test.go

```
@@ -0,0 +1,65 @@
+package analysis
+
+import (
+	"github.com/ahmadateya/flotta-webapp-backend/pkg/logparser"
+	"testing"
+)
+
+func TestGetCPUAvgTempOverTheDay(t *testing.T) {
+	logLines := []logparser.CPUTempParsedLine{
+		{
+			LogDate: "2020-01-01",
+			LogTime: "00:00:00",
+			Data: logparser.CPUTempData{
+				Temp: 50,
+			},
+		},
+		{
+			LogDate: "2020-01-01",
+			LogTime: "00:00:00",
+			Data: logparser.CPUTempData{
+				Temp: 30,
+			},
+		},
+	}
+	avgTemp, _ := GetCPUAvgTempOverTheDay(logLines)
```

Do not ignore errors! Please, add assertion for the returned `err`

---

done